apps/duplication/archive_translate.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+
+from flask import request
+
+from apps.archive.archive import SOURCE as ARCHIVE
+from apps.content import push_content_notification
+from superdesk import get_resource_service
+import superdesk
+from superdesk.errors import SuperdeskApiError, InvalidStateTransitionError
+from superdesk.metadata.item import CONTENT_STATE, ITEM_STATE
+from superdesk.metadata.utils import item_url
+from superdesk.resource import Resource
+from superdesk.services import BaseService
+from superdesk.workflow import is_workflow_state_transition_valid
+
+
+class TranslateResource(Resource):
+    endpoint_name = 'translate'
+    resource_title = endpoint_name
+
+    schema = {
```

Comment:
imo would be better to have `archive_translate` url and put item `guid` into payload, this way we can in theory translate multiple items with single request